Homework OOP library catalog/src/main/java/Library.java

```diff
@@ -0,0 +1,67 @@
+import java.util.Arrays;
+
+public class Library {
+
+    private Book[] books;
+
+    private int getPositionOfBookInList(Book bookToBeSearched) {
+        for (int i = 0; i < books.length; i++) {
+
+            if (books[i].getName().equals(bookToBeSearched.getName())) {
+                return i;
+            }
+        }
+        return -1;
+    }
+
+    private boolean existInLibrary(Book bookToBeSearched) {
+        final int positionOfBookInList = getPositionOfBookInList(bookToBeSearched);
+        return positionOfBookInList != -1;
+    }
+
+    public void addToLibrary(Book book) {
+
+        if (this.books == null) {
```

Comment:
for a much cleaner code, I would recommend to initialise instance variables inside constructor